### ISSUE TITLE
Could not determine required bootstrap compiler version. Should not happen.

### ISSUE THREAD

**Issue by JohnPeel** [LongDirtyAnimAlf/Reiniero-fpcup#38]
```
E:\Programs\fpcup>fpclazup.exe --inifile=settings.ini --inisection=General
Fpclazup, a FPC/Lazarus downloader/updater/installer
Original by BigChimp: https://bitbucket.org/reiniero/fpcup
This version: https://github.com/LongDirtyAnimAlf/Reiniero-fpcup

Fpclazup will download the FPC and Lazarus sources
from the source SVN repositories, and compile, and install.
Result: you get a fresh, up-to-date Lazarus/FPC installation.

Version: based on commit fpcup045 (20171219)
Build date: 2017/12/30 10:40:12
Compiled for CPU: i386 on win32

Options:
Binutils/make dir:  E:\programs\fpcup\bootstrap
Base directory:     C:\development
Bootstrap dir:      E:\programs\fpcup\bootstrap
FPC URL:            http://svn.freepascal.org/svn/fpc/branches/fixes_2_6/
FPC options:        -g -gl -O1
FPC directory:      E:\programs\fpc_2.6
Lazarus URL:        http://svn.freepascal.org/svn/lazarus/branches/fixes_1_4/
Lazarus options:    -g -gl -O1
Lazarus directory:  E:\programs\lazarus_1.4
Keep changes:       no
Re-apply changes:   no
Log file name:      fpcup.log
Standard modules:   suggestedpackages
Add. modules:       FPCCrossWin32-64,LazarusCrossWin32-64,anchordocking

WARNING: FPCUP will download full repositories !!!
FPC is compiled with debug (-g) !
Lazarus is compiled with debug (-g) !

Effective parameters:
--fpcdir=e:\programs\fpc_2.6
--fpcurl=http://svn.freepascal.org/svn/fpc/branches/fixes_2_6/
--fpcopt=-g -gl -O1
--lazdir=e:\programs\lazarus_1.4
--lazurl=http://svn.freepascal.org/svn/lazarus/branches/fixes_1_4/
--lazopt=-g -gl -O1
--primary-config-path=e:\programs\lazarus_1.4\config
--fpcuplinkname=
--binutilsdir=e:\programs\fpcup\bootstrap
--fpcbootstrapdir=e:\programs\fpcup\bootstrap
--include=FPCCrossWin32-64,LazarusCrossWin32-64
--noconfirm
--anchordocking=1
Persistent parameters (can be saved in batch file):
--inifile="settings.ini" --inisection="General"
fpclazup: info: 1/24/2018 1:48:23: fpclazup: V045 (20171219) started.
fpclazup: info: FPCNativeInstaller (CleanModule: FPC): Running make distclean twice (OS_TARGET=invalid/CPU_TARGET=invalid)
fpclazup: info: FPCNativeInstaller (CleanModule: FPC): Execute: E:\programs\fpcup\bootstrap\make.exe. Params: --jobs=16,FPC=E:\programs\fpcup\bootstrap\ppc386.exe,--directory=E:\programs\fpc_2.6,INSTALL_PREFIX=E:\programs\fpc_2.6,UPXPROG=echo,COPYTREE=echo,CPU_SOURCE=i386,OS_SOURCE=win32,CPU_TARGET=i386,OS_TARGET=win32,distclean
fpclazup: info: FPCNativeInstaller (CleanModule: FPC): Execute: E:\programs\fpcup\bootstrap\make.exe. Params: --jobs=16,FPC=E:\programs\fpcup\bootstrap\ppc386.exe,--directory=E:\programs\fpc_2.6,INSTALL_PREFIX=E:\programs\fpc_2.6,UPXPROG=echo,COPYTREE=echo,CPU_SOURCE=i386,OS_SOURCE=win32,CPU_TARGET=i386,OS_TARGET=win32,distclean
fpclazup: info: FPCNativeInstaller (CheckModule: FPC): checking FPC sources with TSVNClient
fpclazup: info: FPCNativeInstaller (CheckModule: FPC): sources ok.
fpclazup: info: FPCNativeInstaller (GetModule: FPC): Start checkout/update of FPC sources.
fpclazup: info: FPCNativeInstaller (DownloadFromSVN: FPC): Running SVN checkout or update.
fpclazup: info: FPCNativeInstaller (GetModule: FPC): FPC was at: branch revision 27788 (repository revision 38026)
fpclazup: info: FPCNativeInstaller (GetModule: FPC): No updates for FPC found.
fpclazup: info: FPCNativeInstaller (GetModule: FPC): No FPC patches defined.
fpclazup: info: FPCNativeInstaller (BuildModule: FPC): Building module FPC...
fpclazup: info: FPCNativeInstaller (BuildModule: FPC): We have a FPC source (@ E:\programs\fpc_2.6) with version: 2.6.5
fpclazup: ERROR: FPCNativeInstaller (BuildModule: FPC): Could not determine required bootstrap compiler version. Should not happen. Aborting.
Fpclazup failed.
Please check log for details. Possible troubleshooting steps:
First action :
Run fpcup again with --verbose and check for make, lazbuild errors etc.
Run fpcup again also in case of incomple download due to SVN errors.

If that does not work ... last resort action :
Remove all files from install-directory, and try again
fpclazup: info: 1/24/2018 1:48:55: fpcup finished.
```

**Comment by JohnPeel** [comment]
2.6.5 doesn't have a REQUIRED_VERSION in it's Makefile.fpc.

**Comment by LongDirtyAnimAlf** [comment]
Confirmed. Will look into it.

**Comment by LongDirtyAnimAlf** [comment]
Will be fixed in next release: 1.6.1b
Fix already in source and tested.
